client/stylesheets/_toCat.scss

```diff
@@ -4996,6 +5003,11 @@ On click the images will be displayed at normal size to complete the effect
     color: tomato !important;
 }
 
+.profileUserFeedbackViewCriteria {
```

Comment:
Selector `profileUserFeedbackViewCriteria` should be written in lowercase with hyphens